targets/TARGET_NORDIC/README.md

```diff
@@ -0,0 +1,55 @@
+# Nordic platform's readme
+
+This readme file contains several hints useful for any Nordic's user.
+
+## 1. Nordic�s development kit targets
+
+Nordic provides several development kit targets, called *[base target]*. For example:
+- NRF51_DK
+- NRF51_DK_LEGACY
+- NRF52_DK
+- NRF51_DONGLE
+- others are incoming as mbed-os is under continuous construction!
```

Comment:
Is it possible to rephrase this sentence in something more positive. It sound like mbed is a loosy product and will always be. Maybe indicate that Nordic supports mbed and will add new targets when new development kits are released.